src/Bundle/DependencyInjection/SyliusResourceExtension.php

```diff
@@ -52,9 +65,31 @@ public function load(array $configs, ContainerBuilder $container): void
         $container->setParameter('sylius.resource.settings', $config['settings']);
         $container->setAlias('sylius.resource_controller.authorization_checker', $config['authorization_checker']);
 
+        $this->autoRegisterResources($config, $container);
+
         $this->loadPersistence($config['drivers'], $config['resources'], $loader);
         $this->loadResources($config['resources'], $container);
 
+        $container->registerForAutoconfiguration(ProviderInterface::class)
+            ->addTag('sylius.state_provider')
+        ;
```

Comment:
```suggestion
        $container
            ->registerForAutoconfiguration(ProviderInterface::class)
            ->addTag('sylius.state_provider')
        ;
```
or 
```suggestion
        $container->registerForAutoconfiguration(ProviderInterface::class)->addTag('sylius.state_provider');
```
if it fits the line characters limit :dancer:

I know I'm meticulous, but it's what makes Sylius Sylius xD